### ISSUE TITLE
Need better contrast

### ISSUE THREAD

**Issue by StevenBlack** [usagi/rust-memory-container-cs#1]
Nice reference!  But it's problematic.

Rationale: A little-less-dark on dark background is never good usability.

I think all instances of text should have higher contrast ratios.  See https://contrast-ratio.com/ for how this is calculated.

The text in the legend has  contrast ratio of just 1.64
https://contrast-ratio.com/#rgb%2858%2C55%2C55%29-on-rgb%2813%2C13%2C13

The text in boxes contrast ratio is just 2.18
https://contrast-ratio.com/#rgb(110%2C105%2C105)-on-rgb(58%2C55%2C55)

This is very basic, but it needs to be emphasized: accessibility and usability matter.  A lot.

![2020-08-21_13-04-49](https://user-images.githubusercontent.com/80144/90917075-29027e80-e3b0-11ea-8ea9-4dd160dada94.jpg)
